examples/vqe2d_lattice.py

```diff
@@ -0,0 +1,81 @@
+import time
+import optax
+import tensorcircuit as tc
+from tensorcircuit.templates.lattice import SquareLattice, get_compatible_layers
+from tensorcircuit.templates.hamiltonians import heisenberg_hamiltonian
+
+# Use JAX for high-performance, especially on GPU.
+K = tc.set_backend("jax")
+tc.set_dtype("complex64")
+# On Windows, cotengra's multiprocessing can cause issues.
+tc.set_contractor("cotengra-8192-8192", parallel=False)
```

Comment:
parallel=False is subtle (very slow), maybe you can try other parallel options to see whether they work in windows? https://cotengra.readthedocs.io/en/latest/advanced.html#parallelization